<?xml version="1.0"?>
<launch>

    <node name="mediapipe_stream" pkg="spot_mediapipe" type="mediapipe_stream.py">
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" />

</launch>
